consultorio-medico/src/main/java/com/iwa/iwaid/consultoriomedico/form/DoctorForm.java

```diff
@@ -0,0 +1,34 @@
+package com.iwa.iwaid.consultoriomedico.form;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+import org.jsondoc.core.annotation.ApiObjectField;
+
+import java.io.Serializable;
+
+@Data
+public class DoctorForm implements Serializable {
+    @ApiObjectField(name="id",description = "Doctor's id")
```

Comment:
los simbolos '=' deben deben de estar separados por espacios para mayor legibilidad
```suggestion
    @ApiObjectField(name = "id", description = "Doctor's id")
```
Replicar en los casos que se repita esta situacion